### ISSUE TITLE
Ship Purchase Error

### ISSUE THREAD

**Issue by rayquaza393** [fvdxd/retribution-src#1]
**If you want a rabbit hole to dive into try and figure out why you cant purchase ships at the dockmaster**

[8:03 PM]
Fri May 03 18:45:18 2024: :UberDogGlobals: mmmm InvItem generated an error
Fri May 03 18:45:18 2024: :UberDogGlobals: tuple index out of range
Fri May 03 18:45:18 2024: ~~Sending :[(51, 8001, 14, 0, 0, 1)] -buying: [[1, 1]] - names; - [('Morning Thunder ', [48, 83])]
Fri May 03 18:45:18 2024: ~~Update Sent!
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Quest
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Purchase
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Repair
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Sell
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Bribe
Fri May 03 18:45:18 2024: DEBUG: InteractGUI.optionName = Exit
[8:03 PM]

**That is the client side error**

[8:08 PM]
:DistributedNPCTownfolkAI: requestMakeShipSale: ([(51, 8001, 14, 0, 0, 1)]) ([]) ([('Iron Runner ', [49, 4])])
:DistributedNPCTownfolkAI(warning): Unable to locate price for shipId: 51
[8:08 PM]

**Server side**

_Basically the DistributedNPCTownfolkAI is not finding the defined ShipID: 51 so it is either defined and/or syntaxed incorrectly or it just was never written by the previous devs_
